modules/preview-react/package.json

```diff
@@ -60,7 +60,6 @@
     "uuid": "^3.3.3"
   },
   "devDependencies": {
-    "@material-ui/core": "^4.9.7",
```

Comment:
We no longer need this after removing `ClickAwayListener` from our examples. Ran a `yarn remove @material-ui/core` from the `preview-react` folder.

Reply:
BYE MUI ✌🏻